opencti-platform/opencti-front/src/private/components/threats/ThreatActorsGroup.jsx

```diff
@@ -133,15 +133,16 @@ class ThreatActorGroups extends Component {
         paginationOptions={paginationOptions}
         numberOfElements={numberOfElements}
         availableFilterKeys={[
+          'x_opencti_workflow_id',
```

Comment:
We can't edit the status field for Group & Individual entities

Reply:
This is weird, because the filter is available in the graphql API, so I just added it here to be consistent with the other filters. I'm going to check again.

Reply:
I am pretty sure this is not relative to your work. We can open an other issue for this.

Reply:
Update : I actually didn't add the filter, I only changed the order, so it's not an issue related to this case.

Reply:
https://github.com/OpenCTI-Platform/opencti/issues/3897